library/src/main/java/com/novoda/downloadmanager/MigrationFactory.java

```diff
@@ -2,39 +2,18 @@
 
 import android.content.Context;
 import android.database.sqlite.SQLiteDatabase;
-import android.os.Handler;
-import android.os.Looper;
 
 import java.io.File;
 
-public final class MigrationFactory {
+final class MigrationFactory {
```

Comment:
This shouldn't need to be exposed to clients of the library...UNLESS we are planning on providing different `Migrators` which is unlikely.

Reply:
Does that mean we can remove the `Migrator` interface? No real need for it if there's only ever going to be one implementation of it